### ISSUE TITLE
Flood Realtime :

### ISSUE THREAD

**Issue by samnawi** [inasafe/inasafe-realtime#92]

**Comment by Charlotte-Morgan** [comment]
@ivanbusthomi  - would you like to see if you can replicate this on the desktop please and confirm what the numbers should be

**Comment by Charlotte-Morgan** [comment]
With the template replication, i would also suggest that we can use some tidier ranges for the symbols since we always have the same population in one area. Maybe <50,000; 50,000 - 100k; 100k - 150 k, 150 - 200k, > 200k (mutually exclusive if possible :)

**Comment by ismailsunni** [comment]
> Refers to the map legend, that circle represents 54K - 109K people affected.
> But, as the information from InaSAFE Analysis result, apparently just 38K people affected.

I think this problem has been fixed already in @lucernae 's PR #93

**Comment by Charlotte-Morgan** [comment]
is it possible to see an example of a report from the #93 ?
Maybe its now showing affected rather than total, but the legend still needs to be fixed to have user friendly breaks and sensible breaks. ie maybe the classes are now silly if we only have 5% of the population displaced ie will we ever see anything bigger than the smallest classes?

@ivanbusthomi - can you please test this on the desktop so we can provide advise to the developers on the required behaviour?

**Comment by ismailsunni** [comment]
Hi @Charlotte-Morgan 
If I checked it in the current realtime, it seems it's already applied. For example http://realtime.inasafe.org/media/reports/flood/pdf/2018010205-6-rw-map.pdf

For the circle, I am still working on that. It should be only a matter of changing the style. I have tried it, but the report is still using the old breakdown.

**Comment by lucernae** [comment]
@ismailsunni do you still have issues by changing the style? IIRC I also did fixed it.

**Comment by ismailsunni** [comment]
The last time I tried, it still didn't get applied to the report. I am not really sure why. The range of the circle is still the same.

**Comment by gubuntu** [comment]
status update @ismailsunni ?

**Comment by ismailsunni** [comment]
I am not working anymore on this issue since my last comment, I checked it, it's still the same: http://realtime.inasafe.org/realtime/flood/impact-map/2018021505-6-rw/en/

**Comment by gubuntu** [comment]
agreed, still seems to be the case @lucernae e.g. http://realtime.inasafe.org/realtime/flood/impact-map/2018021505-6-rw/en/

Also, population symbols are missing from some flooded areas

**Comment by Charlotte-Morgan** [comment]
I think @lucernae - may have fixed this but then it reverted . @lucernae - did you write a new ticket for this or something related recently?

**Comment by lucernae** [comment]
@myarjunar This needs to be fixed in Realtime v4 :D
